src/ServiceProvider.php

```diff
@@ -67,4 +73,19 @@ protected function getAwsClientClosure(string $client, array $config = [])
             return $app->make('aws')->createClient($client, $config);
         };
     }
+
+    private function getLaravelEnvProvider(): callable
```

Comment:
Should be pretty trivial to write some tests for this to keep coverage at 100%

Reply:
Doh, good point :+1: